.goreleaser.yml

```diff
@@ -43,15 +42,14 @@ checksum:
 signs:
   - artifacts: checksum
     args:
-      # if you are using this is a GitHub action or some other automated pipeline, you 
-      # need to pass the batch flag to indicate its not interactive.
       - "--batch"
       - "--output"
       - "${signature}"
       - "--detach-sign"
       - "${artifact}"
-release:
-  # If you want to manually examine the release before its live, uncomment this line:
-  # draft: true
 changelog:
   skip: true
+announce:
+  slack:
+    enabled: true
+    message_template: Succesfully released {{ .ProjectName }} {{.Tag}}
```

Comment:
I thought @tasdomas was asking more about failed release notifications (rather than arguably pointless noisy successful release ones)

Reply:
Feel free to open a pull request reberting this change.

#### Deletions
https://github.com/iterative/terraform-provider-iterative/blob/89ff0415623a98563fcf3fcd495e64a5452b0002/.github/workflows/release.yml#L22
https://github.com/iterative/terraform-provider-iterative/blob/89ff0415623a98563fcf3fcd495e64a5452b0002/.goreleaser.yml#L52-L55

Reply:
* Moved to https://github.com/iterative/terraform-provider-iterative/pull/677